src/tslint-cli.ts

```diff
@@ -207,6 +207,21 @@ const processFile = (file: string) => {
         process.exit(1);
     }
 
+    const buffer = new Buffer(256);
```

Comment:
Done. `readSync` iiuc returns the number of bytes read, but wouldn't overwrite sections of the buffer it doesn't touch.